app/components/entity_mentions/resolution.py

```diff
@@ -14,6 +14,8 @@
     re.IGNORECASE,
 )
 
+CODEBLOCK_REGEX = re.compile(r"```.*?```", re.DOTALL)
```

Comment:
My regex is a little bit rusty, but is this not the same as `.*`?

```suggestion
CODEBLOCK_REGEX = re.compile(r"```.*```", re.DOTALL)
```

Reply:
![86640](https://github.com/user-attachments/assets/b7bc8b85-852b-47f2-8483-4272e9e05c42)

This should probably be using `zig_codeblocks.extract_codeblocks` instead anyway.